lib/commands/location.js

```diff
@@ -35,7 +35,9 @@ commands.setGeoLocation = async function (location) {
   }
   const args = ['-u', this.opts.udid];
   args.push(...formatLocationArg(latitude));
-  args.push(...formatLocationArg(longitude));
+  if (!latitude.startsWith('-')) {
+    args.push(...formatLocationArg(longitude));
+  } else args.push(...(longitude));
```

Comment:
This will break linting, and not be allowed. Please use curly braces (also maybe reverse to get rid of the negation?):
```js
args.push(...formatLocationArg(latitude));
if (latitude.startsWith('-')) {
  args.push(...(longitude));
} else {
  args.push(...formatLocationArg(longitude));
}